src/main/java/net/atos/zac/formulieren/model/FormulierVeldDefinitie.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.formulieren.model;
+
+import static net.atos.zac.util.FlywayIntegrator.SCHEMA;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(schema = SCHEMA, name = "formulier_veld_definitie")
+@SequenceGenerator(schema = SCHEMA, name = "sq_formulier_veld_definitie", sequenceName = "sq_formulier_veld_definitie", allocationSize = 1)
+public class FormulierVeldDefinitie {
+
+    @Id
+    @GeneratedValue(generator = "sq_formulier_veld_definitie", strategy = GenerationType.SEQUENCE)
+    @Column(name = "id_formulier_veld_definitie")
+    private Long id;
+
+    @NotNull
+    @ManyToOne
+    @JoinColumn(name = "id_formulier_definitie", referencedColumnName = "id_formulier_definitie")
+    private FormulierDefinitie formulierDefinitie;
+
+    @NotBlank
+    @Column(name = "systeemnaam", nullable = false, unique = true)
+    private String systeemnaam;
+
+    @NotBlank
+    @Column(name = "volgorde", nullable = false)
+    private int volgorde;
+
+    @NotBlank
+    @Column(name = "label", nullable = false)
+    private String label;
+
+    @NotNull
+    @Column(name = "veldtype", nullable = false)
+    @Enumerated(EnumType.STRING)
+    private FormulierVeldType type;
```

Comment:
Is het dan niet handig om het attribuut zelf ook veldtype te noemen i.p.v. type

Reply:
ja, pas ik aan